flink-ai-flow/QUICKSTART.md

```diff
@@ -2,117 +2,108 @@
+you could start all services with a single scripts. 
 
-If you execute this command for the first time, you will get the following output:
+The mysql database that you created during [Prerequisites](#prerequisites) should be passed as parameter to store metadata of AI Flow
 
-```text
-The ${AIRFLOW_HOME}/airflow.cfg is not exists. You need to provide a mysql database to initialize the airflow, e.g.:
-start-aiflow.sh mysql://root:root@127.0.0.1/airflow
+```shell
+start-aiflow.sh mysql://user:password@ip_or_hostname/airflow
 ```
 
-Please prepare the MySQL database refer to [Prerequisites](#prerequisites-1) and rerun the `start-aiflow.sh` with the MySQL parameter.
 After that you will get the output like:
 
 ```text
-Scheduler log: ${AIRFLOW_HOME}/scheduler.log
-Scheduler pid: 69945
-Web Server log: ${AIRFLOW_HOME}/web.log
-Web Server pid: 69946
-Master Server log:  ${AIRFLOW_HOME}/master_server.log
-Master Server pid: 69947
+Admin user admin created
+Notification service log: ${AIRFLOW_HOME}/logs/notification_service.log
+Notification service pid: 21859
+Scheduler log: ${AIRFLOW_HOME}/logs/scheduler.log
+Scheduler pid: 21860
+Web Server log: ${AIRFLOW_HOME}/logs/web.log
+Web Server pid: 21861
+Master Server log: ${AIRFLOW_HOME}/logs/master_server.log
+Master Server pid: 21862
 Airflow deploy path: ${AIRFLOW_HOME}/airflow_deploy
 Visit http://127.0.0.1:8080/ to access the airflow web server.
 ```
 If you see the login page when visit http://127.0.0.1:8080/, you can login with the defaut user name(admin) and password(admin).
+This is the scheduler web UI which you can see the workflows that you summitted.
 
-### Prepare AI Flow Project
-
-In order to properly adapt to the Airflow, the AI Flow project should have such a directory structure:
-
-```text
-SimpleProject
-├─ project.yaml
-├─ jar_dependencies
-├─ resources
-└─ python_codes
-   ├─ __init__.py
-   ├─ my_ai_flow.py
-   └─ requirements.txt
-```
-
-For python jobs we only need to prepare the `python_codes` directory, the `resources` directory and the `project.yaml`.
-
-### Run Example and Check the Execution Result
+### Run Example
 
-To run the workflow, just execute:
+We have prepared a simple workflow to get started, you could follow several steps to run it.
 
 ```shell
-# This path should be determined by yourself
+# 1. Because the default scheduler service is Apache Airflow, so we need to set ${AIRFLOW_HOME}, this path could be determined by yourself.
 export AIRFLOW_HOME=~/airflow
 AIRFLOW_DEPLOY_PATH="${AIRFLOW_HOME}/airflow_deploy"
 
-# This path is the absolute path of the flink-ai-flow project in your machine
+# 2. The example is prepared in ${SOURCE_CODE_DIR}/examples/demo, ${SOURCE_CODE_DIR} is the absolute path 
+# of the flink-ai-flow project. You can directly copy it to your working directory.
 SOURCE_CODE_DIR=/tmp/flink-ai-extended/flink-ai-flow
+cp -r $SOURCE_CODE_DIR/examples/demo $AIRFLOW_HOME/
```

Comment:
Why does the demo directory move to `AIRFLOW_HOME`?

Reply:
Modified